docs/book/supported_features_matrix.md

```diff
@@ -0,0 +1,20 @@
+<!-- markdownlint-disable MD033 -->
+
+# vSphere CSI Driver - Supported Features Matrix
+
+In the below table - `*` represent features newly available in vSphere 7.0/CSI 2.0
+
+Note: Project pacific and Tanzu Kubernetes Grid support only block volumes.
+
+| Feature                     | Vanilla Block                                                            | Vanilla File   | Project pacific Block | Tanzu K8s Grid for Project pacific Block |
+|-----------------------------|--------------------------------------------------------------------------|----------------|-----------------------|------------------------------------------|
+| Dynamic Provisioning        | Yes                                                                      | Yes `*`        | Yes `*`               | Yes `*`                                  |
+| vSphere UI integration      | Yes                                                                      | Yes `*`        | Yes `*`               | Yes `*`                                  |
+| Multi-master for CSI driver | Yes                                                                      | Yes `*`        | Yes `*`               | Yes `*`                                  |
+| Datastores supported        | vmfs/nfs/vSAN/(vvol `*`)                                                 | vSAN `*`       | vmfs/nfs/vSAN `*`     | vmfs/nfs/vSAN `*`                        |
+| Static Provisioning         | Yes                                                                      | Yes `*`        | No                    | Yes                                      |
+| Access mode                 | RWO                                                                      | ROX,RWX `*`    | RWO*                  | RWO `*`                                  |
+| Volume Topology/Zones       | Yes (beta)                                                               | No             | No                    | No                                       |
+| Extend Volume               | Yes (k8s 1.16+, beta)*                                                   | No             | No                    | No                                       |
```

Comment:
May be we can call out beta only in a separate line so that it's clearer

Reply:
done.